test/e2e/chainsaw/check-arbitary-objects/rgd.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kro.run/v1alpha1
+kind: ResourceGraphDefinition
+metadata:
+  name: check-arbitrary-objects
+spec:
+  schema:
+    apiVersion: v1alpha1
+    kind: CheckArbitraryObjectsNoOp
+    spec:
+      values: object | required=true
```

Comment:
How would the object be used in a resource. Could you provide an example  ?

would it be `spec.values` OR `spec.values.something` ?